service/registry/src/main/java/module-info.java

```diff
@@ -37,6 +37,7 @@
 
     requires transitive io.helidon.builder.api;
     requires transitive io.helidon.common.types;
+    requires java.naming;
```

Comment:
Really?

Reply:
@tomas-langer This is your piece of code.